### PR TITLE
Feat: smooth map movement

### PR DESCRIPTION
**Pull request by fjebaker**
Buggy up and down movement.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,4 +10,5 @@ cart.wasm
 *.toc
 *.tic
 *.vscode
-.DS_Store
\ No newline at end of file
+.DS_Store
+sync.sh
```
Comment (jwiggins) [inline .gitignore]: Stuff that people should have in their local `.gitignore` files 😛

File: cart/player.go [hunk 2]
```diff
@@ -3,19 +3,20 @@ package cart
 import "cart/tic80"
 
 type Player struct {
-	X, Y    int32
-	Frame   int32
-	Sprite  tic80.Sprite
-	Move_fx tic80.SoundEffect
-	Speed   int32
-	Moving  bool
+	X, Y       int32
```
Comment (JR-Mitchell) [inline cart/player.go:6]: Should these be `ScreenX`, `ScreenY`?

Comment (jwiggins) [reply cart/player.go:6]: Probably, yes.

**Review by jwiggins — APPROVED** [review]
Cool. Let's do this.

**Pull request merged**